ipythonblocks/ipythonblocks.py

```diff
@@ -24,6 +24,16 @@
 from IPython.display import HTML, IFrame, display, clear_output
 from IPython.display import Image as ipyImage
 
+# This statement is required for compatibility issues with Python 3.10.
+# Since that version, Iterable and Sequence must be imported from
+# collections.abc, instead of collections
+try:
+    from collections.abc import Iterable
+    from collections.abc import Sequence
+except:
```

Comment:
Let's only catch `ImportError` here so we're only catching that one specific type of error.
```suggestion
except ImportError:
```

Reply:
Done! Thanks a lot for the suggestion 😄!